repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/proxy/ResponseStatusMessageTest.groovy

```diff
@@ -50,6 +50,7 @@ class ResponseStatusMessageTest extends ReposeValveTest {
         mc.receivedResponse.message == message
 
         where:
-        code << (100..599)
+        // todo: add tests for 1xx codes which are currently not supported by Repose
+        code << (200..599)
```

Comment:
It appears that Apache's HTTP Client library treats does not consider `1xx` responses final and awaits a final response. Since Deproxy is only ever sending the `1xx` response, the request times out leading to an exception which eventually leads to a `502` response.

For now, I've just limited the status code range, but we probably want to discuss how to handle `1xx` status codes in the future.